docs/HACKING.md

```diff
@@ -112,47 +189,65 @@ Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
+- The hostname of the instance follows the [RFC 1123](<https://datatracker.ietf.org/doc/html/rfc1123>) DNS label
+standard:
+  - Contain only lowercase alphanumeric characters or '-'.
+  - Start with an alphanumeric character.
+  - End with an alphanumeric character.
+- Create a PTR record corresponding to the instance address which resolves to the instance hostname for successful
+reverse DNS lookups.
+- Uninstall containerd
+- Find the Internal IP address of your cluster
+- find the Windows Administrator username
 
-To deploy machines using the [infrastructure providers supported by WMCO](wmco-prerequisites.md#supported-cloud-providers-based-on-okdocp-version-and-wmco-version),
-refer to the specific provider documentation. See Windows instance [pre-requisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/byoh-instance-pre-requisites.md).
+#### Specify number of Windows instances
 
-The information you need to collect from each Windows instance is:
-- the internal IP address
-- the Windows administrator username
+To run the WMCO e2e tests on a bare metal or other platform-agnostic infrastructure (platform=none), where there is no
+cloud provider specification, the desired number of Windows instances must be provisioned beforehand.
 
-Export `WINDOWS_INSTANCES_DATA` as an environment variable with the corresponding [windows-instances ConfigMap](https://github.com/openshift/windows-machine-config-operator#adding-instances)
-data section, and a new `windows-instances` ConfigMap will be created during the execution of the e2e test suite.
+#### Specify windows instances data
 
-For example:
-```shell
+Find the corresponding [windows-instances ConfigMap](https://github.com/openshift/windows-machine-config-operator#adding-instances)
+and export it as an environment variable. For example:
+
+```shell script
 export WINDOWS_INSTANCES_DATA='
 data:
   10.1.42.1: |-
     username=Administrator
 '
 ```
-where `10.1.42.1` is the IP address and `Administrator` is the Windows' administrator username of the Windows instance.
 
-After the `WINDOWS_INSTANCES_DATA` environment variable is set and exported you can run:
-```shell
-hack/run-ci-e2e-test.sh
+#### Disable deviceless NICs
+
+If the instance uses a static IP, and the instance is not running Windows Server 2022, disable deviceless NICs.
+This can be done by running the following in PowerShell:
+
+```shell script
+New-ItemProperty HKLM:\SYSTEM\CurrentControlSet\Services\hns\State -Name DeviceLessNicDisabled -PropertyType DWORD -Value 1
 ```
 
-## Bundling the Windows Machine Config Operator
-This directory contains resources related to installing the WMCO onto a cluster using OLM.
+## Deploying mimicking the official deployment method
+
+If you want to deploy the WMCO mimicking the official deployment method, first follow the instructions on how to
```

Comment:
Like I mentioned before you are not mimicking but following. In addition this is not official deployment method but the steps for building the artifacts used in the official deployment method.